app/models/concerns/call_listable.rb

```diff
@@ -69,7 +69,7 @@ def clear_call_list(line)
 
   def ordered_patients(line)
     # n+1 join here
```

Comment:
There's more to do here, but we do fix some of the particular egregious cases (and get bullet to hush, so leaving this in.

Reply:
since we've silence the alert, should we throw an issue on the backlog so we don't lose site of it?

Reply:
I think let's keep #2225 open? I'll add a note about what remaining problems there are.